blueocean-github-pipeline/pom.xml

```diff
@@ -26,6 +26,13 @@
     <dependency>
       <groupId>org.kohsuke</groupId>
       <artifactId>github-api</artifactId>
+      <exclusions>
+        <!-- Upper bound fix: Pick the one from jenkins core, it has later version 2.4-->
```

Comment:
This _was_ a bug in `github-api-plugin/pom.xml` but it should be fixed in 1.76. You should *not* have a dependency on the library itself, only on the plugin `org.jenkins-ci.plugins:github-api`. And then you need to make sure that you are consuming Jackson only via `org.jenkins-ci.plugins:jackson2-api`.

Reply:
Fixed, we depend on github-api-plugin.